tests/zfs-tests/tests/functional/acl/off/posixmode.ksh

```diff
@@ -130,7 +130,7 @@ function test_posix_mode # base
 }
 
 # Sanity check on tmpfs first
-tmpdir=$(TMPDIR=$TEST_BASE_DIR mktemp -d)
```

Comment:
The original code makes it easier to clean up the temporary ZTS files after a failed run, since you'd know they'd be in a fixed location (`$TEST_BASE_DIR` aka `/var/tmp`).  I would keep this as-is.

Reply:
Previous commit defaults `TMPDIR` to `FILEDIR`, which is `TEST_BASE_DIR`, making this specific line a no-op change for the default case.

Reply:
Thanks, my mistake, I missed where you added `export TMPDIR="$FILEDIR"`